tests/fixtures/env/teams/jobs.demo.yaml

```diff
@@ -20,22 +20,18 @@ teamConfig:
                       tag: latest
                       pullPolicy: IfNotPresent
                   init:
-                      env:
-                          foo: bar
-                      nativeSecrets:
-                          foo: bar
-                      image:
-                          repository: busybox
-                          tag: latest
-                          pullPolicy: IfNotPresent
-                      script: |
-                          echo "hello $receiver and $SOME-KEY"
-                      secrets:
-                          - mysecret-generic
-                          - hello-otomi
+                      - env:
+                            foo: bar
+                        image:
+                            repository: busybox
+                            tag: latest
+                            pullPolicy: IfNotPresent
+                        script: |
+                            echo "hello $receiver and $SOME-KEY"
```

Comment:
is `$receiver` used in this context?
is `$SOME-KEY` even valid? No, right? Maybe we can use some vars that are injected and valid to not make devs wonder what is going on...

Reply:
Sorry, I just indented the array of containers but we can definitely look at it later?